pkg/graph/schema/types/translation.graphql

```diff
@@ -79,3 +79,109 @@ type TranslationFieldWithOptions {
   options: Map!
 }
 
+"""
+Represents a translation question with no options and a parent
+"""
+type TranslationFieldWithParent {
+  gqlField: String!
+  goField: String!
+  dbField: String!
+  label: String!
+  readonlyLabel: String
+  sublabel: String
+  multiSelectLabel: String
+  isArray: Boolean
+  dataType: TranslationDataType!
+  formType: TranslationFormType!
+  """
+  Is a question a followup to another that doesn't designate it's own readonly question/line
+  """
+  isOtherType: Boolean
+  """
+  Field name for the parent question for fields that represent Other, Please specify, etc.  Used in change history to render parent question for context
+  """
+  otherParentField: String
+  parentRelation: Map!
+}
+
+"""
+Represents a translation question with options and child/children
+"""
+type TranslationFieldWithOptionsAndChildren {
+  gqlField: String!
+  goField: String!
+  dbField: String!
+  label: String!
+  readonlyLabel: String
+  sublabel: String
+  multiSelectLabel: String
+  isArray: Boolean
+  dataType: TranslationDataType!
+  formType: TranslationFormType!
+  """
+  Is a question a followup to another that doesn't designate it's own readonly question/line
+  """
+  isOtherType: Boolean
+  """
+  Field name for the parent question for fields that represent Other, Please specify, etc.  Used in change history to render parent question for context
+  """
+  otherParentField: String
+  options: Map!
+  childRelation: Map!
+}
+
+"""
+Represents a translation question with options and parent
+"""
+type TranslationFieldWithOptionsAndParent {
+  gqlField: String!
+  goField: String!
+  dbField: String!
+  label: String! 
+  readonlyLabel: String
+  sublabel: String
+  multiSelectLabel: String
+  isArray: Boolean
+  dataType: TranslationDataType!
+  formType: TranslationFormType!
+  """
+  Is a question a followup to another that doesn't designate it's own readonly question/line
+  """
+  isOtherType: Boolean
+  """
+  Field name for the parent question for fields that represent Other, Please specify, etc.  Used in change history to render parent question for context
+  """
+  otherParentField: String
+  options: Map!
+  parentRelation: Map!
```

Comment:
Does this need to be a map because of the closure? Since it's 1:1, could it be the Field Name? 

We could maybe also expose the idea of an interface if it's 1:1, and is any type of translation field? I'm good to run with this for this PR and re-visit later!

Reply:
The closure represents a Map basically.  If you look at the export it evaluates to the entire parent type.  May would be better to talk through

Reply:
Sounds good! I'd be happy to sync on this. I think that could potentially be ok to not know the exact parent type, just say it is a translation type (which we could define as a union in the schema). I made essentially this in the Backend, I just didn't expose the type in GQL previously.